nautobot/docs/release-notes/index.md

```diff
@@ -10,6 +10,7 @@
 - UI Component Framework
 - Virtual Device Context Data Models
 - Wireless Data Models
+- Apps Marketplace Page and Installed Apps Page Tile View
```

Comment:
Nit: I think we're alphabetizing these headings both here and in the actual release-note below, for lack of any other more logical ordering.